overlord/configstate/configcore/netplan.go

```diff
@@ -177,8 +177,10 @@ func handleNetplanConfiguration(tr config.Conf, opts *fsOnlyContext) error {
 		return err
 	}
 
-	// Use the default config that most Ubuntu Core have
-	originHint := "90-snapd-config"
+	// Use a origin hint that is sorted before the console-conf
+	// "00-snapd-config.yaml" so that console-conf can override
+	// our settings when it runs.
+	originHint := "0-snap-set"
```

Comment:
sorry, maybe my remark was not clear. We can't just do this either. I fear we need to use different hints at different times:

* 0-snapd-defaults if seeded is false
* 90-snapd-config if seeded is true

Reply:
Sorry for that misunderstanding on my part, updated and tests added (including an early config test).